ingestion/src/metadata/profiler/orm/functions/median.py

```diff
@@ -58,14 +58,24 @@ def _(elements, compiler, **kwargs):
 
 # pylint: disable=unused-argument
 @compiles(MedianFn, Dialects.Athena)
-@compiles(MedianFn, Dialects.Trino)
 @compiles(MedianFn, Dialects.Presto)
 def _(elements, compiler, **kwargs):
     col = compiler.process(elements.clauses.clauses[0])
     percentile = elements.clauses.clauses[2].value
     return "approx_percentile(%s, %.2f)" % (col, percentile)
 
 
+@compiles(MedianFn, Dialects.Trino)
+def _(elements, compiler, **kwargs):
+    col = compiler.process(elements.clauses.clauses[0])
+    percentile = elements.clauses.clauses[2].value
+    return "IF(count(%s) = 0, NULL, approx_percentile(%s, %.2f))" % (
```

Comment:
I think this should be handled differently. If there are no rows in the table, we should just skip. profiling entirely -- but that should be a different PR.